api/src/main/java/com/datatorrent/api/DefaultInputPort.java

```diff
@@ -79,14 +79,15 @@ public void put(T tuple)
     process(tuple);
   }
 
-  /** {@inheritDoc} */
+  /**
```

Comment:
Undo, there is no need to touch this file as it is not  a newly introduced code style violation, there is no any other changes in this file and it is likely to lead to conflict when merging back from the feature branch.